docs/content/users/install/performance.md

```diff
@@ -147,158 +156,175 @@ Instructions for Mutagen and NFS are below.
-    2. Configure the NFS server and exports files using the provided scripts for each operating system.
-    3. Test that NFS is working correctly by using `ddev debug nfsmount` in a project directory. The first line should report something like "Successfully accessed NFS mount of /path/to/project"
-    4. Enable NFS mounting globally with `ddev config global --nfs-mount-enabled`  (You can also configure NFS mounting on a per-project basis with `ddev config --nfs-mount-enabled` in the project directory, but this is unusual. If nfs mounting is turned on globally it overrides any local project settings for NFS.)
-    5. `ddev start` your project and make sure it works OK. Use `ddev describe` to verify that NFS mounting is being used. The NFS status is near the top of the output of `ddev describe`.
+    1. Make sure DDEV is already working and you can use it.
+    2. Use the script below for your OS to configure the NFS server and exports files.
+    3. Test that NFS is working correctly by using `ddev debug nfsmount` in a project directory. The first line should report something like “Successfully accessed NFS mount of /path/to/project”.
+    4. Enable NFS mounting globally with `ddev config global --nfs-mount-enabled`.  
+    You can also configure NFS mounting on a per-project basis with `ddev config --nfs-mount-enabled` in the project directory, but this is unusual. If NFS mounting is turned on globally, it overrides any local project settings for NFS.
+    5. `ddev start` your project and make sure it works normally. Use `ddev describe` to verify that NFS mounting is being used. The NFS status is near the top of the output of `ddev describe`.
 
-    Note that you can use the NFS setup described for each operating system below (and the scripts provided) or you can set up NFS any way that works for you. For example, if you're already using NFS with vagrant on macOS, and you already have a number of exports, the default export here (your home directory) won't work, because you'll have overlaps in your `/etc/exports`. Or on Windows, you may want to use an NFS server other than [Winnfsd](https://github.com/winnfsd/winnfsd), for example the [Allegro NFS Server](https://nfsforwindows.com). The setups provided below and the scripts provided below are only intended to get you started if you don't already use NFS.
+    !!!tip "Skip step 2 if you’re already using NFS!"
+        If you’re already using NFS with Vagrant on macOS, for example, and you already have a number of exports, the default home directory export here won’t work—you’ll have overlaps in your `/etc/exports`. Or on Windows, you may want to use an NFS server other than [Winnfsd](https://github.com/winnfsd/winnfsd) like the [Allegro NFS Server](https://nfsforwindows.com).  
 
-    Note that NFS does not really add to performance on Linux, so it is not recommended.
+        The recommendations and scripts below are for getting you started if you *don’t* already use NFS.
```

Comment:
I have yet to run into anybody who already had NFS deployed for something else...